### PR TITLE
fix: Add gray message to user in dp-box-shadow

### PR DESCRIPTION
**Pull request by GusBaamonde**
#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

messages-user

### CODE REVIEW

**Comment by andresmoschini** [comment]
❤️  No revisé el CSS, pero el concepto se ve muy bien!

**Comment by IgnacioRodrigues** [comment]
(Volví por esta zona) Que bueno Gus, super funcional y reducido el código. 👏

File: assets/scss/modules/_grid-system.scss [hunk 1]
```diff
@@ -79,6 +79,20 @@
...
+      text-decoration: underline;
+      color: $dp-color-red;
+
+      &:hover {
+        color: $dp-color-darkred;
+      }
+    }
+  }
+
+  .dp-boxshadow--msjuser {
```
Comment (cbernat) [inline assets/scss/modules/_grid-system.scss]: How about renaming this class like this: `.dp-boxshadow--usermsg` to keep consistency (most classes are named in english)

**Pull request merged**